content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -374,6 +375,107 @@ Here, `W` is the number of work items.
+Creating a Job with `.spec.suspend` set to true will create it in the suspended
+state.
+
+Remember that suspending a Job will delete all active Pods. When the Job is
+suspended, your [Pods will be terminated](/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination)
+with a SIGTERM signal. The Pod's graceful termination period will be honored and
+your Pod must handle this signal in this period; this may involve saving
+progress for later or undoing changes. Pods terminated this way will not count
+towards the Job's `completions` count.
+
+An example Job definition in the suspended state can be like so:
+
+```shell
+kubectl get job myjob -o yaml
+```
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: myjob
+spec:
+  suspend: true
+  parallelism: 3
+  completions: 15
+  template:
+    spec:
+      ...
+```
+
+The Job's status can be used to determine if a Job is suspended or has been
+suspended in the past:
+
+```shell
+kubectl get jobs/myjob -o yaml
+```
+
+```json
+apiVersion: batch/v1
+kind: Job
+# .metadata and .spec omitted
+status:
+  conditions:
+  - lastProbeTime: "2021-02-05T13:14:33Z"
+    lastTransitionTime: "2021-02-05T13:14:33Z"
+    status: "True"
+    type: Suspended
+  startTime: "2021-02-05T13:13:48Z"
+  succeeded: 3
+```
+
+The Job condition of type "Suspended" with status "True" means the Job is
+suspended; the `lastTransitionTime` field can be used to determine how long the
+Job has been suspended for. If the status of that condition is "False", then the
+Job was previously suspended and is now running. If such a Job condition does
+not exist, the Job has never been stopped.
+
+Events are also created when the Job is suspended and resumed:
+
+```shell
+kubectl describe jobs/myjob
+```
+
+```
+Name:           myjob
+...
+Events:
+  Type    Reason            Age   From            Message
+  ----    ------            ----  ----            -------
+  Normal  SuccessfulCreate  12m   job-controller  Created pod: myjob-rc5fm
+  Normal  SuccessfulCreate  12m   job-controller  Created pod: myjob-cgswm
+  Normal  SuccessfulCreate  12m   job-controller  Created pod: myjob-9vtjs
+  Normal  SuccessfulCreate  11m   job-controller  Created pod: myjob-4v5gk
+  Normal  SuccessfulCreate  11m   job-controller  Created pod: myjob-hlrpl
+  Normal  SuccessfulCreate  11m   job-controller  Created pod: myjob-gmszl
+  Normal  SuccessfulDelete  11m   job-controller  Deleted pod: myjob-gmszl
+  Normal  SuccessfulDelete  11m   job-controller  Deleted pod: myjob-hlrpl
+  Normal  SuccessfulDelete  11m   job-controller  Deleted pod: myjob-4v5gk
+  Normal  Suspended         11m   job-controller  Job suspended
+  Normal  SuccessfulCreate  3s    job-controller  Created pod: myjob-jvb44
+  Normal  SuccessfulCreate  3s    job-controller  Created pod: myjob-vg7x9
+  Normal  SuccessfulCreate  3s    job-controller  Created pod: myjob-tjbgv
+  Normal  Resumed           3s    job-controller  Job resumed
```

Comment:
The way the Job controller works means that the pod creation/deletion events are emitted before status condition-related events. Not sure if the order is an issue even though all of these events are emitted at the same time.